validator/src/main.rs

```diff
@@ -1475,7 +1475,7 @@ pub fn main() {
             Arg::with_name("snapshot_archive_format")
                 .long("snapshot-archive-format")
                 .alias("snapshot-compression") // Legacy name used by Solana v1.5.x and older
-                .possible_values(&["bz2", "gzip", "zstd", "tar", "none"])
+                .possible_values(SUPPORTED_ARCHIVE_COMPRESSION)
                 .default_value("zstd")
```

Comment:
It's brittle that we are using a constant for `SUPPORTED_ARCHIVE_COMPRESSION` but just a local name for the default. We should either make that short-string a constant too, or have a different way to choose a default.

I looked very briefly at it a few days ago, actually, but I haven't had a chance to fix the broken tests. But feels like rather than having this "zstd" we should just implement Default for the ArchiveFormat: #25064

Reply:
Ah. OK. Add a const for default method to avoid using the raw string here.